protobuf-definitions/README.md

```diff
@@ -26,3 +26,14 @@ On Windows & Linux: [See here](https://github.com/google/protobuf/blob/master/sr
 1. Install pre-requisites.
 2. Un-comment line 4 in `make.bat`, and set to correct Grpc.Tools sub-directory.
 3. Run `make.bat`
+4. In the generated `UnityToExternalGrpc.cs` file in the `CommunicatorObjects` folder, you will need to add on top of the file the line
+
+```csharp
+# if UNITY_EDITOR || UNITY_STANDALONE_WIN || UNITY_STANDALONE_OSX || UNITY_STANDALONE_LINUX
+```
+ and at the bottom of the file the line 
+ 
+ ```csharp
+ #endif
+ ```
+This is to make sure the Grpc library is not used on platforms that do not support training.
```

Comment:
Explain why this is needed.

Reply:
You mean more than the current explanation ? What is missing in it ? I say It is to make sure the Grpc library is not used on platforms that do not support training